### PR TITLE
WIP: PWA background push

### PR DESCRIPTION
**Pull request by binwiederhier**

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -26,7 +26,19 @@
...
+    "workbox-cacheable-response": "^6.5.4",
+    "workbox-core": "^6.5.4",
+    "workbox-expiration": "^6.5.4",
+    "workbox-google-analytics": "^6.5.4",
+    "workbox-navigation-preload": "^6.5.4",
+    "workbox-precaching": "^6.5.4",
+    "workbox-range-requests": "^6.5.4",
+    "workbox-routing": "^6.5.4",
+    "workbox-strategies": "^6.5.4",
+    "workbox-streams": "^6.5.4"
```
Comment (binwiederhier) [inline web/package.json:41]: That's a LOT of dependencies. If you can strip that down to what's necessary that'd be great.

**Review by binwiederhier — COMMENTED** [review]
Looks great so far!

**Comment by liamstojanovic** [comment]
I've hit a blocker with this PR. See https://github.com/binwiederhier/ntfy/issues/346

**Comment by binwiederhier** [comment]
@liamstojanovic So I take it this PR is dead?

**Pull request closed without merging**